src/Products/ZCatalog/tests/test_catalog.py

```diff
@@ -325,7 +328,7 @@ def testKeywordIndexWithMinMaxRangeWrongSyntax(self):
         # checkKeywordIndex with min/max range wrong syntax.
         catalog = self._make_one()
         a = catalog(att3={'query': ['att'], 'range': 'min:max'})
-        self.assert_(len(a) != self.upper)
+        self.assertTrue(len(a) != self.upper)
```

Comment:
This line could also use `assertNotEqual`.

Reply:
+1